plugins/topology/README.md

```diff
@@ -190,44 +207,46 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
   Other values result in icons not being rendered for the node.
 
-  > Note: If Red Hat OpenShift Dev Spaces is [installed and configured](https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.7/html/administration_guide/installing-devspaces) and Git URL annotations are also present on the workload YAML file, then clicking on the edit code decorator redirects you to the Red Hat OpenShift Dev Spaces instance.
+- You can also add the `backstage.io/kubernetes-namespace` annotation to identify the Kubernetes resources using the defined namespace.
 
-  > Note: When you deploy the application using the OCP Git import flows, then you do not need to add the labels as import flows add the labels to the workload YAML file. Otherwise, you need to add the labels manually to the workload YAML file.
-
-  The labels are not similar to `backstage.io/edit-url` annotations as it points to the catalog entity metadata source file and is applied to Backstage catalog entity metadata YAML file, but not Kubernetes resources.
+  ```yaml title="catalog-info.yaml"
+  annotations:
+    backstage.io/kubernetes-namespace: <RESOURCE_NS>
+  ```
 
-  > Tip: You can also add the `app.openshift.io/edit-url` annotation with the edit URL that you want to access using the decorator.
+  > The Red Hat OpenShift Dev Spaces instance is not accessible using the edit code decorator if the `backstage.io/kubernetes-namespace` annotation is added to the `catalog-info.yaml` file.
 
-  > Tip: You can use our [prepared manifest for a read-only `ClusterRole`](https://raw.githubusercontent.com/janus-idp/backstage-plugins/main/plugins/topology/manifests/clusterrole.yaml), providing both Kubernetes plugin andd Topology plugin access.
+  To retrieve the instance URL, CheCluster Custom Resource (CR) is required. The instance URL is not retrieved if the namespace annotation value is different from `openshift-devspaces` as CheCluster CR is created in `openshift-devspaces` namespace.
 
-- The following annotation is added to the entity's `catalog-info.yaml` file to identify whether an entitiy contains the Kubernetes resources:
+- A custom label selector is added, which Backstage uses to find the Kubernetes resources. The label selector takes precedence over the ID annotations.
 
   ```yaml title="catalog-info.yaml"
   annotations:
-    backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
+    backstage.io/kubernetes-label-selector: 'app=my-app,component=front-end'
   ```
 
-  You can also add the `backstage.io/kubernetes-namespace` annotation to identify the Kubernetes resources using the defined namespace.
+  If you have multiple entities while Red Hat Dev Spaces is configured and want multiple entities to be able to support the edit code decorator that redirects to the Red Hat Dev Spaces instance. You can add the `backstage.io/kubernetes-label-selector` annotation to the `catalog-info.yaml` file for each entity.
```

Comment:
@debsmita1 above works as expected and does not conflict i.e. if `backstage.io/kubernetes-namespace` annotation is added to the `catalog-info.yaml` file then will the label selector work for entities across namespaces? 

or is it either?

Reply:
When the dev spaces is configured you must not add the namespace annotation to your entity file if you want to access it using the edit code decorator

<img width="1071" alt="Screenshot 2023-09-18 at 4 58 12 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/3726b62d-5f44-46d6-b7a2-e0ac56bc07f2">

If